src/lockTracer.cpp

```diff
@@ -36,6 +36,16 @@ UnsafeParkFunc LockTracer::_orig_unsafe_park = NULL;
 
 
 Error LockTracer::start(Arguments& args) {
+    if (!VM::loaded()) {
+        return Error("lock tracer only works with the JVM loaded");
+    }
```

Comment:
Not needed. This is already checked at a higher level:
https://github.com/async-profiler/async-profiler/blob/75aefa36c9a9c1568c94f3d7e48017b3b3bece1b/src/profiler.cpp#L1095-L1096